Gruntfile.js

```diff
@@ -9,11 +9,11 @@ module.exports = function (grunt) {
         require('./' + task)(grunt);
     });
 
-    grunt.registerTask('test', ['browserify:edge', 'browserify:instrumented', 'mocha', 'coverage-report']);
+    grunt.registerTask('test', ['templates', 'browserify:edge', 'browserify:instrumented', 'browserify:components', 'mocha', 'coverage-report']);
     grunt.registerTask('documentation', ['markdox']);
     grunt.registerTask('validate', ['jshint:all', 'jscs', 'test']);
     grunt.registerTask('concatCometd', ['uglify:cometdMin', 'uglify:cometdDebug']);
-    grunt.registerTask('production', ['concatCometd', 'validate', 'browserify:mapped', 'browserify:min', 'documentation']);
+    grunt.registerTask('production', ['concatCometd', 'validate', 'templates', 'browserify:mapped', 'browserify:min', 'browserify:components', 'copy:components', 'documentation']);
```

Comment:
Can we just register a new task called "components" to combine a few of the different things? Or are they too spread apart?